cmd/release-controller/sync_analysis.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	imagev1 "github.com/openshift/api/image/v1"
+	"k8s.io/klog"
+)
+
+func (c *Controller) ensureAnalysisJobs(release *Release, releaseTag *imagev1.TagReference) error {
+	for name, analysisType := range release.Config.Analysis {
+		if analysisType.Disabled {
+			klog.V(2).Infof("Release analysis step %s is disabled, ignoring", name)
+			continue
+		}
+		jobCount := 10
+		if analysisType.AnalysisJobCount > 0 && analysisType.AnalysisJobCount <= 20 {
+			jobCount = analysisType.AnalysisJobCount
+		}
+
+		// TODO: How do we want to track these jobs...
+		jobLabels := map[string]string{
+			"release.openshift.io/analysis": releaseTag.Name,
```

Comment:
When this job is executed, do we know the payload SHA it originates from?

Reply:
We can probably extract it from `release.Target.Status.Tags[<index>].Items[0].Image`

Reply:
> We can probably extract it from `release.Target.Status.Tags[<index>].Items[0].Image`

I'd like to see that extracted.  That way I can determine identical-ness by looking at prowjob labels.

Reply:
We agreed on a followup